pykickstart/commands/bootloader.py

```diff
@@ -211,7 +205,8 @@ class F12_Bootloader(F8_Bootloader):
 
     def _getParser(self):
         op = F8_Bootloader._getParser(self)
-        op.add_argument("--lba32", dest="forceLBA", deprecated=True, action="store_true")
+        op.add_argument("--lba32", dest="forceLBA", action="store_true",
+                        deprecated=True, version="Fedora12")
```

Comment:
@clumens - I thought a bit about it but left it for later. At the moment I can easily perform some heuristics based on the class name, e.g.
```
version = self.__class__.__name__.replace('FC', 'F').replace('F', 'Fedora').split('_')[0]
```

Is this what you meant or there's more reliable versioning information which I'm not aware of at the moment ?